test/support/application_fixture_helper.rb

```diff
@@ -13,6 +13,11 @@ def setup_application_fixture
     @old_working_dir = Dir.pwd
   end
 
+  def remove_extensions
+    Object.send(:remove_const, :MyLocalExtension) if defined?(MyLocalExtension)
```

Comment:
```suggestion
    Object.send(:remove_const, :MyLocalExtension)
```
If we expect it to be there, it should be there. Otherwise our test is wrong for calling this method.

Reply:
Sure makes sense! Will change.